tests/frontier/scenarios/scenarios/double_call_combinations.py

```diff
@@ -0,0 +1,63 @@
+"""Define Scenario that will run a given program and then revert."""
+
+from typing import List
+
+from ethereum_test_tools import Conditional
+from ethereum_test_tools.vm.opcode import Macro, Opcode
+from ethereum_test_tools.vm.opcode import Macros as Om
+from ethereum_test_tools.vm.opcode import Opcodes as Op
+
+from ..common import Scenario, ScenarioEnvironment, ScenarioGeneratorInput
+
+
+def scenarios_double_call_combinations(scenario_input: ScenarioGeneratorInput) -> List[Scenario]:
+    """
+    Generate Scenarios for double call combinations.
+    First call the operation normally.
+    Then do a subcall that will [OOG,REVERT,RETURN].
+    Second call the operation normally.
+    Compare the results of first call with the second operation call.
+    """
+    scenarios_list: List[Scenario] = []
+    keep_gas = 300000
+    revert_types: List[Opcode | Macro] = [Op.STOP, Om.OOG, Op.RETURN]
+    if Op.REVERT in scenario_input.fork.valid_opcodes():
+        revert_types.append(Op.REVERT)
+    for revert in revert_types:
+        operation_contract = scenario_input.pre.deploy_contract(code=scenario_input.operation_code)
+        subcall_contract = scenario_input.pre.deploy_contract(
+            code=Op.MSTORE(0, 0x1122334455667788991011121314151617181920212223242526272829303132)
+            + revert(0, 32, unchecked=True)
```

Comment:
```suggestion
            + revert(offset=0, size=32)
```
To remove the "unchecked".